flake.nix

```diff
@@ -27,11 +39,25 @@
     {
       packages = {
         rnixLsp = rnixLsp.defaultPackage.${system};
+        inherit nodejs;
         inherit generate-ethereum-account mosh neovim ranger screen tmux;
       };
 
       apps = {
-        generate-ethereum-account = flake-utils.lib.mkApp { drv = self.packages.${system}.generate-ethereum-account; };
+        generate-ethereum-account = flakeUtils.lib.mkApp { drv = self.packages.${system}.generate-ethereum-account; };
+      };
+
+      lib = {
```

Comment:
Is `lib` a new designated flake output?

Reply:
Yes. There is a `lib` convention used with Nix flakes for sharing reusable values.

See https://zero-to-nix.com/concepts/flakes#functions.